### ISSUE TITLE
为什么要最后要再加一个bias呢？

### ISSUE THREAD

**Issue by czisok** [zhougr1993/DeepInterestNetwork#22]
您好，看了你写的代码，在模型训练和预测的时候：self.logits = i_b + d_layer_3_i和elf.logits_sub = tf.sigmoid(item_b[:predict_ads_num] + d_layer_3_sub)都额外加了一个偏差，TensorFlow的dense层不是有bias吗，为什么还有在最后输出的时候再加一个？
希望得到您的解答。

**Comment by zhougr1993** [comment]
因为蠢~

**Comment by sudongxiang** [comment]
> 因为蠢~

同问，这里是怎么加的，本人已经实现了DIN，但是这个bias不知道怎么加。加上这个bias能提升多少？

**Comment by andrew-zzz** [comment]
@sudongxiang @czisok 摆脱进去看看交叉墒的源码 里面有套sigmoid 但并没有加bias

**Comment by czisok** [comment]
> @sudongxiang @czisok 摆脱进去看看交叉墒的源码 里面有套sigmoid 但并没有加bias

Thanks
